### PR TITLE
adjust-ci

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Related-To: https://github.com/thoth-station/metrics-exporter/issues/704

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -11,3 +11,26 @@ presubmits:
             - "pre-commit"
             - "run"
             - "--all-files"
+          resources:
+            requests:
+              memory: "2Gi"
+              cpu: "2"
```
Comment (harshad16) [inline .prow.yaml:17]:
```suggestion
            requests:
              memory: "1Gi"
              cpu: "200m"
            limits:
              memory: "1Gi"
              cpu: "200m"
```

File: .prow.yaml [hunk 2]
```diff
@@ -11,3 +11,26 @@ presubmits:
...
+            - "run-mypy"
+          args:
+            - "."
+            - "--ignore-missing-imports"
+            - "--config-file"
+            - "mypy.ini"
+          resources:
+            requests:
+              memory: "2Gi"
+              cpu: "200m"
```
Comment (harshad16) [inline .prow.yaml:36]:
```suggestion
            requests:
              memory: "1Gi"
              cpu: "200m"
            limits:
              memory: "1Gi"
              cpu: "200m"
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
just some minor required changes.

**Comment by pacospace** [comment]
> @pacospace: The following test **failed**, say `/retest` to rerun all failed tests:
> 
> Test name	Commit	Details	Rerun command
> aicoe-ci/prow/mypy	[386042d](https://github.com/thoth-station/graph-metrics-exporter/commit/386042de66d9059607f9eeca3c372900f1532c0a)	[link](https://prow.operate-first.cloud/view/s3/ci-prow-artifacts--024cd900-706c-488d-afb0-d28522131f45/prow-logs/pr-logs/pull/thoth-station_graph-metrics-exporter/4/thoth-mypy-py38/1405789067928408064)	`/test thoth-mypy-py38`
> [Full PR test history](https://prow.operate-first.cloud/pr-history?org=thoth-station&repo=graph-metrics-exporter&pr=4). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3Apacospace). Please help us and [open an issue](https://github.com/thoth-station/graph-metrics-exporter/issues?q=is:issue+is:open) when you hit one in your PR.

Is this error `Failed to find Pipfile.lock, poetry.lock or requirements.txt in the current directory or any of its parent: /home/prow/go/src/github.com/thoth-station/graph-metrics-exporter` expected from mypy check @harshad16 ?

**Comment by fridex** [comment]
> > @pacospace: The following test **failed**, say `/retest` to rerun all failed tests:
> > Test name	Commit	Details	Rerun command
> > aicoe-ci/prow/mypy	[386042d](https://github.com/thoth-station/graph-metrics-exporter/commit/386042de66d9059607f9eeca3c372900f1532c0a)	[link](https://prow.operate-first.cloud/view/s3/ci-prow-artifacts--024cd900-706c-488d-afb0-d28522131f45/prow-logs/pr-logs/pull/thoth-station_graph-metrics-exporter/4/thoth-mypy-py38/1405789067928408064)	`/test thoth-mypy-py38`
> > [Full PR test history](https://prow.operate-first.cloud/pr-history?org=thoth-station&repo=graph-metrics-exporter&pr=4). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3Apacospace). Please help us and [open an issue](https://github.com/thoth-station/graph-metrics-exporter/issues?q=is:issue+is:open) when you hit one in your PR.
> 
> Is this error `Failed to find Pipfile.lock, poetry.lock or requirements.txt in the current directory or any of its parent: /home/prow/go/src/github.com/thoth-station/graph-metrics-exporter` expected from mypy check @harshad16 ?

I would say so. mypy can expect stub files being present thus it installs requirements. BTW that's also the reason why we do not want to have requirements that are not used in sources stated in Pipfile/Pipfile.lock - they make the testsuite slow and harder to potentionally debug any interference between versions (https://github.com/thoth-station/graph-metrics-exporter/pull/3#discussion_r653456916)

**Comment by pacospace** [comment]
> > > @pacospace: The following test **failed**, say `/retest` to rerun all failed tests:
> > > Test name	Commit	Details	Rerun command
> > > aicoe-ci/prow/mypy	[386042d](https://github.com/thoth-station/graph-metrics-exporter/commit/386042de66d9059607f9eeca3c372900f1532c0a)	[link](https://prow.operate-first.cloud/view/s3/ci-prow-artifacts--024cd900-706c-488d-afb0-d28522131f45/prow-logs/pr-logs/pull/thoth-station_graph-metrics-exporter/4/thoth-mypy-py38/1405789067928408064)	`/test thoth-mypy-py38`
> > > [Full PR test history](https://prow.operate-first.cloud/pr-history?org=thoth-station&repo=graph-metrics-exporter&pr=4). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3Apacospace). Please help us and [open an issue](https://github.com/thoth-station/graph-metrics-exporter/issues?q=is:issue+is:open) when you hit one in your PR.
> > 
> > 
> > Is this error `Failed to find Pipfile.lock, poetry.lock or requirements.txt in the current directory or any of its parent: /home/prow/go/src/github.com/thoth-station/graph-metrics-exporter` expected from mypy check @harshad16 ?
> 
> I would say so. mypy can expect stub files being present thus it installs requirements. BTW that's also the reason why we do not want to have requirements that are not used in sources stated in Pipfile/Pipfile.lock - they make the testsuite slow and harder to potentionally debug any interference between versions ([#3 (comment)](https://github.com/thoth-station/graph-metrics-exporter/pull/3#discussion_r653456916))

Thanks @fridex! Then we need to merge the other PR to have Pipfile

**Review by fridex — APPROVED** [review]
You can use `git pull --rebase` to avoid unnecessary merge commits in the git history. Also, having one PR per feature can make the git history clean and more maintainable - this will be helpful for future debugging.

thanks 👍🏻

**Pull request merged**